Validation/RecoEgamma/plugins/ElectronMcSignalValidator.cc

```diff
@@ -3805,6 +3836,7 @@ void ElectronMcSignalValidator::analyze(const edm::Event &iEvent, const edm::Eve
     // match distributions
     h1_ele_EoP->Fill(bestGsfElectron.eSuperClusterOverP());
     h2_ele_EoPVsEta->Fill(bestGsfElectron.eta(), bestGsfElectron.eSuperClusterOverP());
+    h2_ele_EoPVsEtaExtended->Fill(bestGsfElectron.eta(), bestGsfElectron.eSuperClusterOverP());
```

Comment:
You are duplicating the previous histo, only with a slightly wider x-range: is it really what you want?
One single histo with the widest range cannot suit as well?

Reply:
Hi Andrea,
yes, it is intended. The idea is to introduce a new category of plots to monitor electrons in 2.5<|eta|<3 without affecting the rest of the plots. Florian

Reply:
Hi Andrea,
Thanks for your understanding. When we will add additional histos to monitor the electrons with |eta|>2.5, we will get rid of these two histograms. Florian